ansible/roles/test/tasks/iface_naming_mode/show_pfc_counters.yml

```diff
@@ -0,0 +1,32 @@
+- block:
+    - name: get pfc Rx counter values 
+      shell:  sudo show  pfc  counters | sed   -n '/Port Rx/,/^$/p' 
+      register: pfc_rx
+
+    - debug: var=pfc_rx.stdout_lines
+
+    - name: get pfc Tx counter values
+      shell:  sudo show  pfc  counters | sed   -n '/Port Tx/,/^$/p'
+      register: pfc_tx
+
+    - debug: var=pfc_tx.stdout_lines
+
+    - name: check if the output shows default interface names in default mode 
+      assert:
+        that:
+          - "'{{item}}' in pfc_rx.stdout and '{{item}}' in pfc_tx.stdout "
+          - "'{{port_name_map[item]}}' not in pfc_rx.stdout  and  '{{port_name_map[item]}}' not in  pfc_tx.stdout"
```

Comment:
I feel that `port_name_map` and `port_alias_map` are confusing names. I think they would be more clear if renamed `name_to_alias_map` and `alias_to_name_map`.

Reply:
The variables  port_name_map and port_alias_map are used from already existing library port_alias.py. 
https://github.com/Azure/sonic-mgmt/blob/master/ansible/library/port_alias.py

And these variables are already in use by few scripts like ,
-	ansible/roles/test/tasks/link_flap/link_flap_helper.yml
-	ansible/roles/fanout/templates/sonic_deploy.j2

Do you want to rename the variable names  in the existing library  and also the scripts which uses it.

Reply:
Since the change would touch many files, I think it is more appropriate to make those changes in a separate PR.